faq/obo-file-format-support.md

```diff
@@ -0,0 +1,62 @@
+
+
+
+# Support for OBO File Format
+A number of ontologist develop their ontologies using the [OBO file format](https://owlcollab.github.io/oboformat/doc/GO.format.obo-1_4.html). This page provides support for maintaining OBO file ontologies in accordance with OBO Foundry principles.   
```

Comment:
Not sure how to do a PR on this PR, so I will just suggest improvements to the wording.
A number of ontologist develop -> Many ontologists develop
principles -> (add link)